src/amber/controller/base.cr

```diff
@@ -13,11 +13,15 @@ module Amber::Controller
     protected getter raw_params : HTTP::Params
     protected getter context : HTTP::Server::Context
     protected getter params : Amber::Validators::Params
+    protected getter flash : Amber::Router::Flash::Hash
```

Comment:
Hash continues to exist everywhere in it's original state. I'm creating Fash::Hash and nothing operates within that namespace. So hash would never be overwritten.